src/promptflow/promptflow/_sdk/_pf_client.py

```diff
@@ -134,10 +135,12 @@ def visualize(self, runs: Union[List[str], List[Run]]) -> None:
 
     @property
     def runs(self) -> RunOperations:
+        """Run operations that can manage runs."""
         return self._runs
 
     @property
     def connections(self) -> ConnectionOperations:
+        """Connection operations that can manage connections."""
         return self._connections
 
     @property
```

Comment:
remove the chat with chat flow part of the docstirng

Reply:
Removed.